src/services/lnurlService.ts

```diff
@@ -123,27 +123,27 @@ const getLnurlAddressParams = async (lnurlAddress: string) => {
 
 
 
-const getInvoice = async(lnurlParams: LNURLPayParams, amountMsat?: number) => {
+const getInvoice = async (lnurlParams: LNURLPayParams, amountMsat?: number, comment?: string) => {
+    if (!amountMsat || amountMsat === 0) { amountMsat = lnurlParams.minSendable }
 
-    if(!amountMsat || amountMsat === 0) {
-        amountMsat = lnurlParams.minSendable
-    }
-
-    const url = lnurlParams.callback.includes('?') ? `${lnurlParams.callback}&amount=${amountMsat}` : `${lnurlParams.callback}?amount=${amountMsat}`
+    const url = new URL(lnurlParams.callback)
```

Comment:
👍

Reply:
yeah i was looking at it and though, wait a minute, this can be done nicely with `URLSearchParams` :)